secureli/services/language_support.py

```diff
@@ -402,21 +438,23 @@ def _write_pre_commit_configs(
         non_success_warnings = list[str]()
 
         # if successfully loaded any language specific configs
-        for config in linter_config:
-            try:
-                for key in config:
-                    config_name = f"{slugify(language)}.{key}.yaml"
-                    path_to_config_file = Path(f".secureli/{config_name}")
-
-                    with open(path_to_config_file, "w") as f:
-                        f.write(yaml.dump(config[key]))
-
-                    num_configs_wrote += 1
-            except Exception as e:
-                num_configs_non_success += 1
-                non_success_warnings.append(
-                    f"Unable to install config: {config_name}. {e}"
-                )
+        for language_linters in linter_configs:
```

Comment:
too much nesting here.  Needs to be broken down into functions with descriptive names.

Reply:
Changed to parsing through an declared object instead of an anonymous array of garbage. Should make it more clear on whats happening.